packages/blockchain-wallet-v4-frontend/src/data/components/swap/test-utils/makeAccount.ts

```diff
@@ -0,0 +1,25 @@
+import { CoinType } from '@core/types'
+import { SwapBaseCounterTypes } from 'data/components/swap/types'
+import { SwapAccountType } from 'data/types'
+
+export const makeAccount = (): SwapAccountType => ({
+  balance: '200000',
+  baseCoin: 'BTC',
+  coin: 'BTC' as CoinType,
+  label: 'Trading Wallet',
+  type: SwapBaseCounterTypes.ACCOUNT
+})
+export const makeBaseAccount = (): SwapAccountType => ({
+  balance: '200000',
+  baseCoin: 'BTC',
+  coin: 'BTC' as CoinType,
+  label: 'Trading Wallet',
+  type: SwapBaseCounterTypes.ACCOUNT
+})
+export const makeCounterAccount = (): SwapAccountType => ({
+  balance: '200000',
+  baseCoin: 'ETH',
+  coin: 'ETH' as CoinType,
+  label: 'Trading Wallet',
+  type: SwapBaseCounterTypes.ACCOUNT
+})
```

Comment:
Can we just use `makeAccount` and amend the return value in the test itself (e.g. as in `proceedToSwapConfirmation.test.ts`)? `BTC` is base account only for one particular case.